Server-side requests to the API, event.context.cloudflare got undefined

### Environment

```
- Operating System: Darwin
- Node Version:     v22.11.0
- Nuxt Version:     3.16.2
- CLI Version:      3.25.0
- Nitro Version:    2.11.9
- Package Manager:  pnpm@9.12.3
- Builder:          -
- User Config:      compatibilityDate, devtools, nitro, modules
- Runtime Modules:  nitro-cloudflare-dev@0.2.2
- Build Modules:    -
```

### Reproduction

https://github.com/dongnaebi/nuxt-cf

### Describe the bug

In my `/api/home`, I attempt to read `event.context.cloudflare.env.DB` to connect to the database. Everything works fine during local debugging with `npm run dev (nuxt dev)`. However, when using `npm run preview (npm run build && wrangler dev)`, the value obtained from `event.context.cloudflare` is `undefined`.

```
// ...
Your worker has access to the following bindings:
- D1 Databases:
  - DB: test-nuxt-cf (6ba1c928-d86a-48ff-966f-0e1057767b94) [simulated locally]
- R2 Buckets:
  - R2: test-nuxt-cf [simulated locally]
[wrangler:inf] Ready on http://localhost:59102
⎔ Starting local server...
event.context.cloudflare:  undefined
event.context.cloudflare:  undefined
✘ [ERROR] [request error] [unhandled] [GET] http://localhost:59102/api/home

   TypeError: Cannot read properties of undefined (reading 'env')

```

What's puzzling is that right after restarting the service, accessing `/api/home` directly works fine. But if I first visit the homepage (which triggers a server-side request to `/api/home`), it results in an error (the error is logged in the console but doesn't appear on the page).

Steps to reproduce:
```bash
pnpm i
# works fine
npm run dev

npm run preview
# visit / and check out console
```

I tried upgrading and downgrading Nitro, and even deploying to the Cloudflare production environment, but none of these solutions resolved the issue.

### Additional context

_No response_

### Logs

```sh

```

--- a/src/runtime/internal/app.ts
+++ b/src/runtime/internal/app.ts
@@ -75,6 +75,7 @@ function createNitroApp(): NitroApp {
           };
       if (fetchContext?._platform) {
         event.context = {
+          _platform: fetchContext?._platform, // #3335
           ...fetchContext._platform,
           ...event.context,
         };

